sprintQuovantis/sprintParam/views.py

```diff
@@ -37,42 +43,117 @@ def post(self, request):
                 status=status.HTTP_400_BAD_REQUEST)
         except Exception as e:
             logging.error(e)
+            print(e)
             return Response(
                 {
                     "message": "Data not stored"
                 },
                 status=status.HTTP_400_BAD_REQUEST)
 
+    @verify_token
     def get(self, request):
         """
         this method is created for retrieve data
         :param request: format of the request
         :return: Response
         """
         try:
-            sprint = Sprint.objects.filter(Sprint_id=request.data.get("Sprint_id"))
+            sprint = Sprint.objects.all()
             serializer = SprintSerializer(sprint, many=True)
+            print(sprint)
+            print(serializer.data)
             return Response(
                 {
                     "message": "Here your sprint",
                     "data": serializer.data
                 },
-                status=status.HTTP_201_CREATED)
+                status=status.HTTP_200_OK)
         except Exception as e:
             print(e)
             logging.error(e)
             return Response(
                 {
-                    "message": "No notes for you"
+                    "message": "No sprint for you"
+                },
+                status=status.HTTP_400_BAD_REQUEST)
+
+    @verify_token
+    def put(self, request, id):
+        """
+
+        :param request: format of the request
+        :param id: sprint id
+        :return: response
+        """
+        print("hello")
+        try:
+            sprint = Sprint.objects.get(id=id)
+            print(id)
+            serializer = SprintSerializer(sprint, data=request.data)
+            serializer.is_valid(raise_exception=True)
+            serializer.save()
+            return Response(
+                {
+                    "message": "Sprint updated successfully",
+                    "data": serializer.data
+                },
+                status=status.HTTP_202_ACCEPTED)
```

Comment:
202 request accepted is for asychronous/batch requests where immediate response is not expected as some background task is running. Here, you should return 200 ok for PUT requests.